packages/react-next/src/components/OverflowSet/OverflowButton.base.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { KeytipManager } from 'office-ui-fabric-react/lib/utilities/keytips/KeytipManager';
+import { IKeytipProps } from '../../Keytip';
+import { IProcessedStyleSet } from '../../Styling';
+import {
+  IOverflowSetItemProps,
+  IOverflowSetProps,
+  IOverflowSetStyles,
+  IOverflowSetStyleProps,
+} from './OverflowSet.types';
+import { classNamesFunction } from '../../Utilities';
+import { useConst } from '@uifabric/react-hooks';
+
+interface IOverflowInternalState {
+  persistedKeytips: { [uniqueID: string]: IKeytipProps };
+  wrapperDivProps: React.HTMLProps<HTMLDivElement>;
+  overflowKeytipSequences: string[] | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  newOverflowItems: any[];
+}
+
+const getClassNames = classNamesFunction<IOverflowSetStyleProps, IOverflowSetStyles>();
+
+const useKeytipRegistrations = (
+  persistedKeytips: { [uniqueID: string]: IKeytipProps },
+  keytipManager: KeytipManager,
+) => {
+  React.useEffect(() => {
+    Object.keys(persistedKeytips).forEach((keytipId: string) => {
+      const keytip = persistedKeytips[keytipId];
+      const uniqueID = keytipManager.register(keytip as IKeytipProps, true);
+      // Update map
+      persistedKeytips[uniqueID] = keytip;
+      delete persistedKeytips[keytipId];
+    });
+    return () => {
+      // Delete all persisted keytips saved
+      Object.keys(persistedKeytips).forEach((uniqueID: string) => {
+        keytipManager.unregister(persistedKeytips[uniqueID] as IKeytipProps, uniqueID, true);
+        delete persistedKeytips[uniqueID];
+      });
+    };
+  }, [persistedKeytips, keytipManager]);
+};
+
+export const OverflowButtonBase = (props: IOverflowSetProps) => {
+  const keytipManager: KeytipManager = KeytipManager.getInstance();
+  const { className, styles, vertical, overflowItems, keytipSequences } = props;
+  const classNames: IProcessedStyleSet<IOverflowSetStyles> = getClassNames(styles, { className, vertical });
+
+  const internalState = useConst<IOverflowInternalState>(() => ({
+    persistedKeytips: {},
```

Comment:
This does need to be persisted, but since it's the only piece of persisted data (see comments below) you can just do `const persistedKeytips = useConst</*type here*/>({});` and get rid of the state wrapper.